### DISCUSSION TITLE
Ganache installation problem

### DISCUSSION THREAD

**Discussion by mustafaassaad** [ConsenSys-archive/ganache#3724] [General]
i'm having trouble with ganache. I keep getting (i installed ganache too) "zsh: command not found: ganache" after trying to fork.url. I'm following it in your tutorial at 1:00:00 https://www.youtube.com/watch?v=LJGCkYC2ZCE&t=3622s

output after running "npm i -g ganache":

<img width="1440" alt="Screen Shot 2022-09-27 at 6 54 01 am" src="https://user-images.githubusercontent.com/89585671/192380175-2e00dca8-f86f-45f2-be26-805b5545180e.png">

**Comment by MicaiahReid** [comment]
Can you try deleting the directories mentioned in the error before installing?
```console
rm -rf /Users/mariamstar1/.nvm/versions/node/v16.15.1/lib/node_modules/ganache
rm -rf /Users/mariamstar1/.nvm/versions/node/v16.15.1/lib/node_modules/.ganache-GSa2jWcg
npm i ganache -g
```
Also, as a side note, copied text from the console would be preferred over screenshots for future responses. Thank you for the question!
